### DISCUSSION TITLE
hse_kvs_cursor_seek_range inclusive vs exclusive filters

### DISCUSSION THREAD

**Discussion by victorstewart** [hse-project/hse#627] [Q&A]
per a cursory look, it appears that `filt_max` of `hse_kvs_cursor_seek_range` is inclusive per https://github.com/hse-project/hse/blob/ca2bccd60e29a74f2e8b587a9b8d4702c360865c/lib/c0/c0sk.c#LL1119C5-L1119C5 ? and thus i assume `filt_min` is also inclusive?

i think it would be great to note these in the header documentation.

**Comment by tristan957** [comment]
https://github.com/hse-project/hse/blob/master/include/hse/hse.h#L1117

On this line, `[@p filt_min, @p filt_max]` is supposed to indicate an inclusive range.

**Comment by tristan957** [comment]
I will also let you know that the team maintaining HSE all got laid off. The future of the project is in limbo. I have thought about forking it, but without all the testing on hardware that we used to do, I am not sure how viable continued development could become.
